v2.0/openapi-v2.json

```diff
@@ -4664,6 +4701,22 @@
             "in": "query",
             "required": false,
             "type": "string"
+          },
```

Comment:
This API is changed to support folders, files or web_links. could you please take out this change. We could address it as part of the another issue?